easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -51,9 +53,20 @@ def extra_options(extra_vars=None):
         extra_vars.update({
             'srcdir': [None, "Source directory location to provide to cmake command", CUSTOM],
             'separate_build_dir': [False, "Perform build in a separate directory", CUSTOM],
+            'buildtype': [CMAKE_BUILD_TYPES[0], "Build type for CMake. Accepted values: " + ', '.join(CMAKE_BUILD_TYPES),
+                          CUSTOM],
         })
         return extra_vars
 
+    def __init__(self, *args, **kwargs):
+        """Initialize generic CMake easyblock."""
+
+        super(CMakeMake, self).__init__(*args, **kwargs)
+
+        if self.cfg['buildtype'] not in CMAKE_BUILD_TYPES:
+            raise EasyBuildError("The specified build type for CMake is not known. Accepted values: %s",
+                                 ', '.join(CMAKE_BUILD_TYPES))
+
```

Comment:
OK, then this check doesn't make sense, indeed, we should allow the build type to be specified freely, and simply use `Release` as default.